conference.js

```diff
@@ -1637,9 +1637,27 @@ export default {
 
         if (!this.localPresenterVideo && !mute) {
             // create a new presenter track and apply the presenter effect.
-            const { height } = this.localVideo.track.getSettings();
-            const defaultCamera
-                = getUserSelectedCameraDeviceId(APP.store.getState());
+            let { height } = this.localVideo.track.getSettings();
+
+            // Workaround for Firefox since it doesn't return the correct width/height of the desktop stream
+            // that is being currently shared.
+            if (!height || height === 0) {
```

Comment:
can `height` here ever be undefined? if not, the first part of the check covers all possibilities, doesn't it?

Reply:
Done